File: Modul 0/Soal 4/pr/PRAK004-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>
#include <string.h>

const int ROW = 35;
const int COLLUMN = 6;

int main(){
    char nama[] = "Muhammad Guntur Ricky Adhitya";
    char nim[] = "2410817310003";
    int row = ROW;
    int collumn = COLLUMN;


    for(int i = 0; i < collumn; i++){
        
        for(int j = 0; j < row; j++){
            if (i == 0  || i+1 == collumn)
            {
                printf("#");
            }
            else if (j == 0 || j+1 == row)
            {
                
                printf("#");
            }
            else{
                if(i == ((collumn/2)-1) & j == ((row-strlen(nama))/2)){
                    printf("%s", nama);
                    row = row - strlen(nama);
                }
                if(i == ((collumn/2)) & j == ((row-strlen(nim))/2)){ 
                    printf("%s", nim);
                    row = row - strlen(nim);
                }
                printf(" ");
            }
            
        }
        row = ROW;
        printf("\n");
    }
    getchar();
    return 0;
}

File: Modul 0/Soal 4/pr/PRAK004-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
ROW = 43
COLLUMN = 8

nama = "Muhammad Guntur Ricky Adhitya"
nim = "2410817310003"

row = ROW
collumn = COLLUMN

i = 0
while i < collumn :
        j = 0
        while j < row : 
            if i == 0 or i+1 == collumn :
                print("#", end="")
            elif j == 0 or j+1 == row :
                print("#", end="")
            else:
                if i == (int((collumn/2))-1) and j == (int((row-len(nama))/2)):
                    print(nama, end="")
                    row = row - len(nama)
                if i == (int((collumn/2))) and j == (int((row-len(nim))/2)):
                    print(nim, end="")
                    row = row - len(nim)
                print(" ", end="")
            j += 1
            
        row = ROW
        print("\n", end="")
        i += 1